armory/docker/management.py

```diff
@@ -6,7 +6,7 @@
 
 import docker
 
-from armory import paths
+from armory import paths, scenarios
```

Comment:
This import is causing the host tests to fail (in conjunction with the updated init in scenarios).

Let's move the `END_SENTINEL` to somewhere like "armory/common.py"